modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpers.fixture';
+import {clientExtensionsPageTest} from '../../fixtures/clientExtensionsPageTest';
+import {ClientExtensionsPage} from '../../pages/product-navigation-applications-menu/ClientExtensionsPage';
+
+export const test = mergeTests(apiHelpersTest, clientExtensionsPageTest);
+
+test('Editor Config Contributor CET is disabled if FF is set to false', async ({
+	_apiHelpers,
+	clientExtensionsPage,
```

Comment:
It was odd to me to see `_` prefix in a prop. I'm assuming no underscore is also another convention that we can apply for consistency.

Reply:
Yeah, I don't like it either. And the playwright docs don't use it. Don't know why it is used... :thinking: